claims-data/service/src/main/resources/db/migration/V6__create_claims_table.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE claims (
+    id                                       UUID PRIMARY KEY,
+    submission_id                            UUID NOT NULL REFERENCES submissions(id),
+    status                                   TEXT NOT NULL CHECK (status IN ('READY_TO_PROCESS', 'NOT_VALIDATED', 'INVALID', 'VALID')),
```

Comment:
Comparing with the Data Dictionary, this seems to be missing the value "REPLACED"